core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerService.scala

```diff
@@ -95,7 +95,8 @@ class LoadBalancerService(
     info(this, s"blackboxFraction = $blackboxFraction")
 
     /** We run this often on an invoker before going onto the next. */
-    private val activationCountBeforeNextInvoker = 10
+    private val activationCountBeforeNextInvoker = Math.max(1, config.loadbalancerActivationCountBeforeNextInvoker.toInt)
```

Comment:
@markusthoemmes @perryibm we've done this in a few places because the WhiskConfig is not type safe - I'm wondering though why we don't move `toInt` into the getter as a first step?

(@RSulzmann you may ignore my comment for this pr)